conference/src/main/java/org/spine/samples/lobby/conference/ConferenceServiceImpl.java

```diff
@@ -0,0 +1,206 @@
+        if (!persistedConference.getIsPublished()) {
+            final Conference publishedConference = persistedConference.toBuilder()
+                                                                      .setIsPublished(true)
+                                                                      .build();
+
+            sendConferencePublishedEvent(publishedConference);
+            final PublishConferenceResponse response = PublishConferenceResponse.newBuilder()
+                                                                                .setId(persistedConference.getId())
+                                                                                .build();
+            responseObserver.onNext(response);
+            responseObserver.onCompleted();
+
+        } else {
+
+            //TODO:2016-04-19:andrii.loboda: handle if conference is already published
+        }
+
+    }
+
+    protected BoundedContext getBoundedContext() {
+        //TODO:2016-04-14:andrii.loboda:  move to singleton and provide event bus, command bus, etc.
+        return BoundedContext.newBuilder()
+                             .build();
+    }
+
+    protected ConferenceRepository getRepository() {
+        return new ConferenceRepository();
+    }
+
+
+    private void sendConferenceCreatedEvent(Conference createdConference) {
+        final ImmutableList.Builder<Event> result = ImmutableList.builder();
+        final ConferenceCreated conferenceCreatedEvent = EventFactory.conferenceCreated(createdConference);
+        final EventContext conferenceEventContext = EventUtils.createConferenceEventContext(createdConference.getId());
+        result.add(Events.createEvent(conferenceCreatedEvent, conferenceEventContext));
+
+        sendEvents(result.build());
+    }
+
+    private void sendConferenceUpdatedEvent(Conference updatedConference) {
+        final ImmutableList.Builder<Event> result = ImmutableList.builder();
+        final ConferenceUpdated conferenceUpdatedEvent = EventFactory.conferenceUpdated(updatedConference);
+        final EventContext conferenceEventContext = EventUtils.createConferenceEventContext(updatedConference.getId());
+        result.add(Events.createEvent(conferenceUpdatedEvent, conferenceEventContext));
+//TODO:2016-04-19:andrii.loboda: looks weird
+        sendEvents(result.build());
+    }
+
+    private void sendConferencePublishedEvent(Conference publishedConference) {
+        final ImmutableList.Builder<Event> result = ImmutableList.builder();
+        final ConferencePublished conferencePublishedEvent = EventFactory.conferencePublished(publishedConference);
+        final EventContext conferenceEventContext = EventUtils.createConferenceEventContext(publishedConference.getId());
+        result.add(Events.createEvent(conferencePublishedEvent, conferenceEventContext));
+
+        sendEvents(result.build());
+    }
+
+
+    private void sendEvents(Iterable<Event> eventsToSend) {
```

Comment:
I think it's better to name the param just `events` to be laconic. Because it is obvious that we want to send them. Also, please rename the method to `postEvents` to use Ubiquitous Language.